stable/yourls/templates/deployment.yaml

```diff
@@ -0,0 +1,138 @@
+      containers:
+        - name: {{ template "fullname" . }}
+          image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          env:
+            - name: ALLOW_EMPTY_PASSWORD
+            {{- if .Values.allowEmptyPassword }}
+              value: "yes"
+            {{- else }}
+              value: "no"
+            {{- end }}
+            - name: YOURLS_DB_HOST
+            {{- if .Values.mariadb.enabled }}
+              value: {{ template "mariadb.fullname" . }}:3306
+            {{- else }}
+              value: {{ .Values.externalDatabase.host | quote }}
+            {{- end }}
+            - name: YOURLS_DB_NAME
+            {{- if .Values.mariadb.enabled }}
+              value: {{ .Values.mariadb.db.name | quote }}
+            {{- else }}
+              value: {{ .Values.externalDatabase.database | quote }}
+            {{- end }}
+            - name: YOURLS_DB_USER
+            {{- if .Values.mariadb.enabled }}
+              value: {{ .Values.mariadb.db.user | quote }}
+            {{- else }}
+              value: {{ .Values.externalDatabase.user | quote }}
+            {{- end }}
+            - name: YOURLS_DB_PASS
+              valueFrom:
+                secretKeyRef:
+                {{- if .Values.mariadb.enabled }}
+                  name: {{ template "mariadb.fullname" . }}
+                  key: mariadb-password
+                {{- else }}
+                  name: {{ printf "%s-%s" .Release.Name "externaldb" }}
+                  key: db-password
+                {{- end }}
+            - name: YOURLS_DB_PREFIX
+              value: {{ .Values.yourlsTablePrefix | quote }}
+            - name: YOURLS_SITE
+              value: {{ .Values.yourlsSite | quote }}
+            - name: YOURLS_USER
+              value: {{ .Values.yourlsUsername | quote }}
+            - name: YOURLS_PASS
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "fullname" . }}
+                  key: yourls-password
+          {{- if .Values.extraEnv }}
+{{ toYaml .Values.extraEnv | indent 12 }}
+          {{- end }}
+          ports:
+            - name: http
+              containerPort: 80
+            - name: https
+              containerPort: 443
+          livenessProbe:
+            httpGet:
+              path: /admin/
+            {{- if not .Values.healthcheckHttps }}
+              port: http
+            {{- else }}
+              port: https
+              scheme: HTTPS
+            {{- end }}
+{{ toYaml .Values.livenessProbe | indent 10 }}
+          readinessProbe:
+            httpGet:
+              path: /admin/
+            {{- if not .Values.healthcheckHttps }}
+              port: http
+            {{- else }}
+              port: https
+              scheme: HTTPS
+            {{- end }}
+{{ toYaml .Values.readinessProbe | indent 10 }}
+          volumeMounts:
```

Comment:
Sorry I have another request. I was working with your chart and wanted to enable some plugins that require external config. Id propose following the convention of allowing the release to define custom volume and volumemounts. It would allow mounting the plugin config directly into the path where it is needed.

Heres an example from Datadog:

Volumes:
https://github.com/helm/charts/blob/69f25df0f3732d9047d713108756ad272592ac6d/stable/datadog/templates/deployment.yaml#L135-L137

VolumeMounts:
https://github.com/helm/charts/blob/69f25df0f3732d9047d713108756ad272592ac6d/stable/datadog/templates/deployment.yaml#L104-L106

Reply:
Sure, let me add this! 😄